contributors/guide/release-notes.md

```diff
@@ -32,6 +40,20 @@ For pull requests that don't need to be mentioned at release time, use the `/rel
     NONE
     ```
 
-To see how to format your release notes, view the kubernetes/kubernetes [pull request template](https://git.k8s.io/kubernetes/.github/PULL_REQUEST_TEMPLATE.md) for a brief example. Pull Request titles and body comments can be modified at any time prior to the release to make them friendly for release notes.
+Ideally, your release note should be written in simple, easy to understand sentences. Users and the team that curates release notes are typically much less familiar with the deep technical implementation details included in your PR. When writing a release note, consider what _users_ need to know and write it so they can easily understand.
```

Comment:
Maybe we can shorten this a bit as follows?

`Your release note should be written in clear and straightforward sentences. Most often, users aren't familiar with the technical details of your PR, so consider what they _need to know_ when you write your release note.`

Reply:
Great! Thanks :smile: